docs/intl.md

```diff
@@ -0,0 +1,155 @@
+# Spark Regional Date Time Support
+
+## Spark Components support regional date functionality
+
+- Spark components have been updated to support regional date functionality in latest versions of **V3** and **V4**.
+- The new updates in Spark are internally feature toggled so that we can release the new date/time regional at the same time for all apps
+- There are two methods for enabling the new regional date features in Spark components: a global variable, and a feature toggle in the form of a cookie
+  o To test the new Spark component regional date formatting in your application, set the global variable on the window object:
+
+      ```
+      window['GUX_OPTIONS_enableRegionalDates']= true
+      ```
```

Comment:
Feel free to disregard but it might be useful to include details of the cookie here too just because it can be useful when trying it out on the examples.